.github/workflows/build.yml

```diff
@@ -43,11 +43,17 @@ jobs:
         if: runner.os == 'Windows'
         uses: ilammy/setup-nasm@v1
 
+      - name: Install ninja-build tool for aws-lc-fips-sys on Windows
+        if: runner.os == 'Windows'
+        uses: seanmiddleditch/gha-setup-ninja@v4
+
       - name: cargo build (debug; default features)
         run: cargo build --locked
 
-      - name: cargo test (debug; all features)
-        run: cargo test --locked --all-features
+      # nb. feature sets that include "fips" should be --release --
+      # this is required for fips on windows.
```

Comment:
It's ok to perform a dev/debug build on Windows with the FIPS feature enabled. The `aws-lc-fips-sys` build-script maps a dev/debug build (i.e. `OPT_LEVEL=0`) [to a "relwithdebinfo" build for AWS-LC](https://github.com/aws/aws-lc-rs/blob/f31bee9052926ed38ef6efc71ffaf54170c9d5fb/aws-lc-fips-sys/builder/cmake_builder.rs#L97).